### PR TITLE
PLAT-294 stepGetObjectState with delay

### PR DESCRIPTION
**Pull request by aalekseev-ru**
**- What I did**
add waiting before sending VStateReport
**- How I did it**
InitReasonCTMethod now jumps into stepWaitState
then if 10% of pulse time is done and object is not ready we send VStateReport once
**- How to verify it**
run tests

### CODE REVIEW

File: ledger-core/v2/virtual/object/object.go [hunk 1]
```diff
@@ -152,12 +157,16 @@ func (sm *SMObject) Init(ctx smachine.InitializationContext) smachine.StateUpdat
 		return ctx.Stop()
 	}
 
+	waitDuration := time.Second * time.Duration(sm.pulseSlot.PulseData().NextPulseDelta) / 10
```
Comment (ivanshibitov) [inline ledger-core/v2/virtual/object/object.go:160]: May be move 10% to config?

Comment (aalekseev-ru) [reply ledger-core/v2/virtual/object/object.go:160]: @ivanshibitov which one? I knew only configuration.go, but it's more about components, not constants. I can move that to object SM constant

Comment (bigbes) [reply ledger-core/v2/virtual/object/object.go:160]: Just move it to constant, for now

File: ledger-core/v2/virtual/integration/object_test.go [hunk 2]
```diff
@@ -57,7 +57,13 @@ func TestInitViaCTMethod(t *testing.T) {
 
 	select {
 	case <-requestIsDone:
-	case <-time.After(10 * time.Second):
+		require.Failf(t, "", "SM Object needs to wait until sm.waitGetStateUntil")
```
Comment (bigbes) [inline ledger-core/v2/virtual/integration/object_test.go:60]: Please, add unit test that will check for this.

Moreover, that test may be flaky. If system scheduler will pause thread executing goroutine - we'll accept result sooner and it'll fail.

File: ledger-core/v2/virtual/object/object.go (hunk 1 above)
Comment (bigbes) [reply ledger-core/v2/virtual/object/object.go:160]: I'd suggest to move that part to separate function (or method)

**Pull request merged**